A clean up of several loose ends.

JessSumner opened the pull request:
Why:
The app contained several files which were unnecessary. Some features were clumped together. The fonts are not uniform across the app.

This PR:
Extracts the fonts being used into the variables file. All headers are now the same font and the rest of the text is another.
Routes have been cleaned up and unecessary routes and pages have been deleted. All routes are RESTful now.
Company logos added to the vet resource cards which did not display a map.
Other small formatting changes to even out the spacing between features.

JessSumner committed:
Clean up and CSS tweaks

Why:
Adding in bitters brought default styling. Resource cards were not clear as text was too large. Overall, just wanted to customize the product more.

How:
Removed ; from the adoption_application.js. This was a recommended change however it broke the function.
Imported a new font "Playfair Display". This was used for the main body of text.
Changed line height to 1.25em. This helps with the text to fit on the resource cards.
Added extra padding-left to the resource card <p> so that it is inline with the title of the card.
Text with link to adoption application has been given the class text so that it aligns with the other text on the page.
Margin-bottom for the nav-bar has been increased to give more space between the nav-bar and headers
Used columns to position the nav menu which is more sustainable than previous method.

JessSumner committed:
Removed redundancies and cleaned up routes

Why:
Small, simple and consistent is better. I want the app to be RESTful and easy to read and adapt.

This commit:
Moved CSS for the home page to it's own file.
Deleted about.html.erb as the content is included on the home page.
The view for the home page is now homes/show.html.erb
Generated a homes controller with a show action.
Renamed the routes so that the root is now homes#show.
Deleted extra routes that are not needed at this time.

JessSumner committed:
Font uniformity and pictures added on some resource cards.

Why:
Some vet resource cards don't have maps due to being mobile or having multiple locations. Their cards look empty it would be nice to add something there.

This commit:
Pulls the two fonts into the variables file in base.
All headers are Architects Daughter.
All the rest of the text is Playfair Display.
The margin top and bottom for resources section headers were increased and decreased respectively to give more space.
The company logos were added to the resource cards with no maps.
An if/else statement was included so if a map attribute was present the view would create the iframe for the map otherwise it would look for a vetlogo. This prevented an empty iframe from appearing on the resource card.

JessSumner committed:
Aligning vet logos to the right of the resource cards.

Why:
Having the logos aligned to one side looks more uniform. The right side of the card is emptier in most cases.

This commit:
Uses the float property to align the logo to the right of the card.
Padding has then been used to adjust the logo's position to look less abrupt.

JessSumner commented:
@MattMSumner and @kenyonj I have more changes if you have time. Thank you

kenyonj commented on app/views/adoption_applications/new.html.erb:
```diff
@@ -102,7 +104,7 @@
     
     <div class="Submit">
-        <%= f.submit "Submit Adoption Application"%>
+      <%= f.submit "Submit Adoption Application"%>
```
This is a pretty long form. Thoughts on extracting it into a partial called `_form_fields.html.erb` ? That might make this view template a little less daunting.

JessSumner committed:
Hound corrections for clean up

This commit:
Removes empty lines from routes.
Removes an extra semicolon from base/variables
Adds line preceeding rule declarations
Reduces selector depth to 2 for home.scss
Orders properties alphabetically.

JessSumner committed:
Javascript fix and new partial for adoption application

Why:
The js for the adoption application form was only working when the page was refreshed but not when the route was followed from home to the form. The view of the adoption application is very information heavy.

This PR:
Updates the js so that the event triggers on every page load.
The bulk of the adoption application has been moved to a partial _form_fields.html.erb. This clears the view so it is less overwhelming and has more clarity on it's purpose.
Deleted some trailing whitespace and ordered some css properties.

MattMSumner commented:
:+1: :shipit:
